src/libslic3r/PrintConfig.cpp

```diff
@@ -1562,6 +1562,12 @@ void PrintConfigDef::init_fff_params()
     def->mode = comAdvanced;
     def->set_default_value(new ConfigOptionFloat(0.));
 
+    def = this->add("brim_compense_efc", coBool);
+    def->label = L("Compense brim EFC gap");
+    def->tooltip = L("Takes into account the Elephant Foot Compensation for calculating the gap, ensuring adhesion between the brim and the part regardless of said compensation.");
```

Comment:
"Takes into account the Elephant Foot Compensation for calculating the gap, shifting the brim inwards accordingly. Usually not needed. Check the wiki for more information."

I would write the tooltip this way since, when EFC is used as such, adhesion takes place even if a gap is left empty.

I will take care of the wiki in the next days.